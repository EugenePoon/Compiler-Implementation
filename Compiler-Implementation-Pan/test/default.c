int main()
{
    int i = 0;
    int n = 10;
    switch(n) {
        case 0: print i+0;
        case 1: print i+1;
        case 2: print i+2;
        default: print n*n;
             
    }
}