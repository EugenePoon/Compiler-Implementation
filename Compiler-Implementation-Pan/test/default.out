24 19 0 5 25 13 0 0 12 13 0 1 1 0 10 12 13 0 1 1 11 18 30 13 11 22 15 -1 16 40 13 0 1 1 11 0 1 6 17 50 13 11 0 1 1 22 15 -1 16 60 13 0 1 1 11 0 2 6 17 70 13 11 0 2 1 22 15 -1 16 83 13 0 1 1 11 13 0 1 1 11 6 17 97 13 0 1 1 11 13 0 1 1 11 3 22 21 2 21 1